### ISSUE TITLE
Rule Proposal: recommend baseUrl

### ISSUE THREAD

**Issue by alecxe** [alecxe/eslint-plugin-protractor#37]
This is to warn if absolute URLs are used in `browser.get()`. Recommend setting `baseUrl` instead.

Invalid usage:

``` js
browser.get("https://google.com/search")
```

Valid usage:

``` js
browser.get("/search")
```

**Comment by alecxe** [comment]
Closed by 325301b61d30ede9c31340e1ed992f17071e652c.
